docs/knowledge_base/demos/Churn Classification.md

```diff
@@ -0,0 +1,31 @@
+In the fast-paced world of business, retaining customers is a top priority. The Churn Classification demo, powered by Taipy, is here to revolutionize customer retention. In this demo, we'll explore this multi-page application that encompasses data visualization, model management, model comparison, and predictive analytics.

```

Comment:
A link from "Churn Classification" to something relevant (such as https://en.wikipedia.org/wiki/Customer_attrition) would be nice.